README.md

```diff
@@ -284,7 +300,7 @@ of all services that are accessible from your local machine
 
 ### ensure-kubectl
 Makes sure kubectl is installed and available for use. Customize the version
-by specifying the `KUBECTL_VERSION` envrionmental variable. Default: `v1.3.6`.
+by specifying the `KUBECTL_VERSION` envrionmental variable. Default: `v1.4.0`.
```

Comment:
It does not. It can accept a version string but 1.4.0 is not an available version.
They do their own versioning and I'm not sure what version of the gcloud package will include the 1.4.0 `kubectl`I can find it if we want to go that route but we don't need to install the gcloud binary, the standard binary works well too.  Maybe we just pull the gcloud component install of kubectl out and install from the link?